walrus/api.py

```diff
@@ -9,20 +8,52 @@ def hello():
     return "Hello World"
 
 
-def get_capital(country_name, raise_errors=False):
-    # Load the data
-    data = pd.read_csv(file_path)
+def find_capital(country_name, raise_errors=False):
+    # Dictionary to store country and capital as key-value pair
+    country_capital = {}
 
-    # Find the row where the 'CountryName' column matches the input country_name
-    country_data = data[data['CountryName'] == country_name]
+    # Open the csv file with the reading mode
+    with open(file_path, 'r') as file:
+        # Split the file line by line and store it in a list
+        data = file.readlines()
+
+        # Header of the csv file
+        header = data[0].split(",")
+
+        # Indexes for country and capitals, respectively
+        country_col = header.index('CountryName')
+        capital_col = header.index('CapitalName')
+
+        # Ignore the first line
+        # Store country and its capital as a key-value pair
+        for line in data[1:]:
```

Comment:
One of the +ves about Python is that code can be expressive with very few lines of code - this matters in large projects. Let's see if we can do this entire for loop in a single line of code.

Reply:
Appended the entire for loop to a single line of code, but separated them into different lines anyway to follow flake8 convention.

Reply:
I think we can do without having to do `line.split(",")` twice.